memory/memory_manager.py

```
@@ -587,12 +596,12 @@ async def _hippocampus_process(self, session: str, chunks: list):
 
         群聊走双路径（个人 + 群组分别提取），私聊走单路径。
         """
-        if not self._llm_client:
-            # LLM client 是延迟注入接口（set_llm_client）—— "插件刚启动 → 用户
+        if not self._extraction_client and not self._reflection_client:
+            # LLM client 是延迟注入接口（set_llm_clients）—— "插件刚启动 → 用户
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)